src/routes/PlayQuiz/index.tsx

```diff
@@ -255,10 +255,10 @@ export default function PlayQuiz() {
               const winners = getWinners(gameInfo.teams);
 
               await updateGame({
-                gameId,
+                gameId: parseInt(gameId as string),
```

Comment:
Why gameId is string?

Reply:
gameId is coming from path param soo it will be a string, we have to do parseInt
https://github.com/rajatkantinandi/quizmaster/pull/23/files#diff-f1a38f7be0969b046489bbd7d5a2fb2cb84b13076255af0e62e4d165dec8e082R33

Reply:
OK got it.